tests/unit/test_model_record.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+
+class TestModelRecord:
+    @pytest.fixture
+    def testRecord(self, mocker):
+        mockHasVersion = mocker.MagicMock()
+        mockHasVersion.hex = 'first edition'
+        return mocker.MagicMock(
```

Comment:
I don't think we need to use `MagicMock` here. We should just create a plain `Record` object and then run the tests on it. I _think_ this should trigger the getter/setter functionality.

Right now I don't think this is testing the actual model, it's only ever reaching mocked objects/methods. The tests themselves look good, so I think this is just about making sure they reach the actual code we need to test!